### PR TITLE
[Term Entry] JavaScript Dates .getUTCDay() #2338 

### PR DESCRIPTION
**Pull request by leonly-wd**
### Description

-Added term entry for the method getUTCDay for JavaScript

### Type of Change

- Adding a new entry 

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/dates/terms/getUTCDay/getUTCDay.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+Description: 'Returns the day of the week in the specified date according to universal time.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-developer-career-path'
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getUTCDay/getUTCDay.md:12]:
```suggestion
  - 'paths/front-end-engineer-career-path'
```

File: content/javascript/concepts/dates/terms/getUTCDay/getUTCDay.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+---
+
+The ** `getUTCDay()` ** method returns the day of the week in the specified date according to universal time, as an integer value from 0 - 6 where 0 represents Sunday, 1 for Monday and so on.
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getUTCDay/getUTCDay.md:15]:
```suggestion
The **`getUTCDay()`** method returns the day of the week in the specified date according to universal time. The returned value is an integer from `0` to `6` where `0` represents Sunday, `1` for Monday and so on.
```

File: content/javascript/concepts/dates/terms/getUTCDay/getUTCDay.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+## Syntax
+
+```javascript
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getUTCDay/getUTCDay.md:19]:
```suggestion
```pseudo
```

File: content/javascript/concepts/dates/terms/getUTCDay/getUTCDay.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+getUTCday();
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getUTCDay/getUTCDay.md:20]:
```suggestion
date_obj.getUTCday();
```

File: content/javascript/concepts/dates/terms/getUTCDay/getUTCDay.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+```
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getUTCDay/getUTCDay.md:21]: add a line break between the block and the sentence

File: content/javascript/concepts/dates/terms/getUTCDay/getUTCDay.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+The return value will give you a integer between 0 and 6:
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getUTCDay/getUTCDay.md:22]:
```suggestion
The return value of the passed `Date` object will yield an integer between `0` and `6`:
```

File: content/javascript/concepts/dates/terms/getUTCDay/getUTCDay.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+## Example
+
+The following example shows the method in use.
+
+```javascript
+const date = new Date('10 June 2023 14:15:30');
+console.log(date.getUTCDay());
+```
+
+The output will be 6.
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getUTCDay/getUTCDay.md:40]:
```suggestion
The output will be:

```shell
6
```
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@leonly-wd I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by leonly-wd** [comment]
@caupolicandiaz Hi I've made the changes as requested but I think I made a mistake by adding an extra commit. There should only be one commit the Update getUTCDay file commit.

**Review by caupolicandiaz — APPROVED** [review]
@leonly-wd I've gone ahead and made some edits that were outstanding and removed the duplicate file. Please review the changes carefully for future reference.

**Comment by leonly-wd** [comment]
> @leonly-wd I've gone ahead and made some edits that were outstanding and removed the duplicate file. Please review the changes carefully for future reference.

Ok thank you

**Comment by SSwiniarski** [comment]
@leonly-wd Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/javascript/dates/getUTCDay

You're currently listed as an anonymous contributor. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.

image

To add your GitHub username to your Codecademy profile:

1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:

image

2.) Make sure to set your visibility to “everyone”:

image

3.) Then log out and log back into Codecademy using your GitHub login:
 
image

Note: Your email used for Codecademy will have to be listed in your GitHub profile for the login feature to properly work. If you 
use the same email for both GitHub and Codecademy already, then you're set.

In addition, we may reach out in the future if any adjustments to these entries are required.

**Pull request merged**